common/lib/xmodule/xmodule/course_module.py

```diff
@@ -303,6 +303,12 @@ class CourseFields(object):
         help=_("Enter the unique identifier for your course's video files provided by edX."),
         scope=Scope.settings
     )
+    facebook_url = String(
+        help=_("Enter the URL for the official course Facebook group. If you provide a URL, the edX mobile app includes a button that students can tap to access the group."),
```

Comment:
Per Lou, it is intended for a Facebook Group. There is no restriction, however: you can put in a Facebook Page URL if you want to. 
Something to ponder is, we have no control over what Facebook labels the structures they allow one to create, or when they choose to change the labels to something else.